hawc/apps/eco/models.py

```diff
@@ -535,68 +146,49 @@
 
 class Country(models.Model):
 
-    country = models.CharField(
-        verbose_name="Country",
-        max_length=100,
-        choices=country_choices,
-        help_text="Select one or more countries",
-    )
+    code = models.CharField(blank=True, max_length=2)
+    name = models.CharField(unique=True, max_length=64)
 
     def __str__(self):
-        return self.country
+        return self.name
 
     class Meta:
         verbose_name = "Country"
+        verbose_name_plural = "Countries"
+        ordering = ("name",)
 
 
 class State(models.Model):
 
-    state = models.CharField(
-        verbose_name="State",
-        max_length=100,
-        choices=state_choices,
-        blank=True,
-        help_text="Select one or more states, if applicable",
-    )
+    code = models.CharField(blank=True, max_length=2)
+    name = models.CharField(blank=True, unique=True, max_length=64)
 
     def __str__(self):
 
-        return self.state
+        return self.name
 
     class Meta:
         verbose_name = "State"
 
 
 class Climate(models.Model):
 
-    climate = models.CharField(
-        verbose_name="Climate",
-        max_length=100,
-        choices=climate_choices,
-        blank=True,
-        help_text="Select one or more climates to which the evidence applies",
-    )
+    name = models.CharField(max_length=100, blank=True)
 
     def __str__(self):
 
-        return self.climate
+        return self.name
 
     class Meta:
         verbose_name = "Climate"
 
 
 class Ecoregion(models.Model):
 
-    ecoregion = models.CharField(
-        verbose_name="Ecoregion",
-        max_length=100,
-        choices=ecoregion_choices,
-        blank=True,
-        help_text="Select one or more Level III Ecoregions, if known",
-    )
+    name = models.CharField(verbose_name="Ecoregion", max_length=100, blank=True,)
 
     def __str__(self):
-        return self.ecoregion
+        return self.name
 
     class Meta:
         verbose_name = "Ecoregion"
```

Comment:
Sorry, I can't quite reach the `class Reference` model, but we should remove this model, and use a `study.Study` model in all places where it's referenced.  Also, is it always true that there's only one Metadata for a Study, or is that a ForeignKey relation

Reply:
Done.